_posts/2023-09-28-kimai-mobile-ios-release.md

```diff
@@ -0,0 +1,75 @@
+
+![Kimai Mobile DarkMode]({% link /images/news/Kimai-Mobile-DarkMode.png %}){:class="image"}
+
+### Track anywhere with offline mode 
+
+* Uninterrupted tracking: Experience time tracking freedom through Offline Mode. 
+* No connectivity worries: No more concerns about connectivity – log your work hours during flights, remote locations, or instances of internet issues. 
+
+![Kimai Mobile Offline]({% link /images/news/Kimai-Mobile-Offline.png %}){:class="image"}
+
+### Take a break: Mindfulness for enhanced productivity 
+
+* Prioritize well-being: Kimai Mobile's break reminders prioritize your mental health and productivity. 
+* Stay refreshed: Friendly notifications encourage timely breaks, allowing you to stay revitalized and maintain peak efficiency throughout your day. 
+
+![Kimai Mobile Break]({% link /images/news/Kimai-Mobile-Break.png %}){:class="image"}
+
+### Data-driven insights with comprehensive reports 
+
+* Empower with insights: Kimai mobile's statistical analysis provides a comprehensive overview of your time usage across diverse projects. 
+* Customized reports: Tailor time frames to receive detailed reports, enabling you to make informed decisions. 
+
+![Kimai Mobile Reports]({% link /images/news/Kimai-Mobile-Reports.png %}){:class="image"}
+
+### In conclusion: Empower your time management with Kimai Mobile 
+
+To conclude, Kimai Mobile surpasses being a mere time tracking app; it's your dedicated partner for impeccable time management. With its multi-language support, Dark Mode, Offline Mode, Break reminders, and data-rich reports, Kimai Mobile arms you with the tools for efficient time tracking and amplified productivity. 
+
+Seize the opportunity to optimize your time management and supercharge productivity wherever you go. Immerse yourself in the transformative experience by trying Kimai Mobile today and witness the profound difference it can make in your professional life. 
+
+Ready to Take Charge? 
+
+![Kimai Mobile Download Now]({% link /images/news/Kimai-Mobile-DownloadNow.png %}){:class="image"}
+
+Are you poised to take command of your time like never before? Embark on your journey with Kimai Mobile and unlock the realm of efficient time tracking. 
+
+Download Kimai Mobile now:  
+
+[Google Play](https://play.google.com/store/apps/details?id=de.cloudrizon.kimaimobile&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1)
+
+[App Store](https://apps.apple.com/us/app/kimai-mobile/id1563933145)
+
+Stay connected with the Kimai community and remain updated with the latest features and tips: [Join the Kimai Community](https://www.kimaimobile.com/the-kimai-mobile-blog/)
+
+For regular updates, follow us on social media: 
+
+*[Pinterest](https://www.pinterest.de/kimaimobileapp/)
+
+*[Facebook](https://www.facebook.com/cloudrizon/)
+
+*[LinkedIn](https://www.linkedin.com/company/cloudrizon/)
```

Comment:
you can add them to your developer profile at /data/developers.yml

remove them from here